results/cstr_results_30_dantzig.csv

```diff
@@ -785,4 +785,83 @@ D-SDA,k=2,baron,3.1301626261862903,20.72,optimal,15.91499992646277,29
 D-SDA,k=Infinity,knitro,2.769569717001105,21.69,optimal,1.4819982228800694,30
 D-SDA,k=2,knitro,3.13019813302529,4.63,optimal,0.32499886583536897,30
 D-SDA,k=Infinity,baron,2.769566949670231,840.51,optimal,820.9910024190322,30
-D-SDA,k=2,baron,3.1301981330248903,17.78,optimal,13.41500028502196,30
\ No newline at end of file
+D-SDA,k=2,baron,3.1301981330248903,17.78,optimal,13.41500028502196,30
```

Comment:
Why are there now two different solutions in this single file?

Reply:
I added the Complete Enumeration to visualize the cstr comparison. I would erase this part from this graph and add another csv file for visualization.